src/main/java/com/registration/model/User.java

```diff
@@ -1,19 +1,40 @@
 package com.registration.model;
 
+import org.springframework.data.annotation.Id;
+
 import com.fasterxml.jackson.annotation.JsonInclude;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
 import lombok.Data;
 
 @Data
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class User {
+	@Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     private int sid;
+	@NotBlank(message = "UserName is mandatory")
     private String username;
+	@NotBlank(message = "First Name is mandatory")
     private String firstName;
+	@NotBlank(message = "Last Name is mandatory")
     private String lastName;
+	@NotBlank(message = "Email is mandatory")
```

Comment:
@NotBlank-  message - Email is required
@Email - message - Please enter valid email